localstack/utils/docker_utils.py

```diff
@@ -917,6 +917,12 @@ def _build_run_create_cmd(
 class Util:
     MAX_ENV_ARGS_LENGTH = 20000
 
+    @staticmethod
+    def format_env_vars(key: str, value: Optional[str]):
+        if value is None:
+            return key
```

Comment:
nit: Just to confirm - looks like if we return `key`, then the env var will not be exposed to the container at all, whereas if we add `f"{key}="` to the `--env-file` file, then it would get added to the environment as an empty string. I'm assuming we want the former behavior here, right? (so no need for a change..)

Reply:
You are right. When only key is passed to docker (also reproducable with `docker exec -e KEY env`) this environment variable is not passed to the process, even if defined in `docker create`. This is necessary since the check in lambci is basically if the value is there, so best thing seems like just deleting it.